commit 881d149a7fa8122deb1da135bd132c0603e6385e
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Wed Mar 29 16:08:29 2017 +0100

    bcm2835-sdhost: mmc_card_blockaddr fix
    
    Get the definition of mmc_card_blockaddr from drivers/mmc/core/card.h.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/drivers/mmc/host/bcm2835-sdhost.c b/drivers/mmc/host/bcm2835-sdhost.c
index a9bc79bfdbb7..9c6f199a7830 100644
--- a/drivers/mmc/host/bcm2835-sdhost.c
+++ b/drivers/mmc/host/bcm2835-sdhost.c
@@ -51,8 +51,12 @@
 #include <linux/of_dma.h>
 #include <linux/time.h>
 #include <linux/workqueue.h>
+#include <linux/interrupt.h>
 #include <soc/bcm2835/raspberrypi-firmware.h>
 
+/* For mmc_card_blockaddr */
+#include "../core/card.h"
+
 #define DRIVER_NAME "sdhost-bcm2835"
 
 #define SDCMD  0x00 /* Command to SD card              - 16 R/W */
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1128_881d149a7fa8122deb1da135bd132c0603e6385e.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1128_881d149a7fa8122deb1da135bd132c0603e6385e.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
